### PR TITLE
Add more spices into indian spices cheatsheet

### PR DESCRIPTION
**Pull request by hiteshaswani121**
https://duck.co/ia/view/indian_spices_cheat_sheet

Maintainer: @shahbaz17

### CODE REVIEW

**Comment by daxtheduck** [comment]
@hiteshaswani121 Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/indian-spices.json [hunk 1]
```diff
@@ -101,6 +102,33 @@
                 "val" : "अज़वाइन",
                 "trn" : "Ajwain"
             }            
+        ],
+        "Spices-IV" : [
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/indian-spices.json]: Are there any better categories we could use to group these?

Comment (shahbaz17) [inline share/goodie/cheat_sheets/json/indian-spices.json]: These categories are just to have a good style when displayed on full.
We are looking for a suitable category or else we can design the stylesheet.

@hiteshaswani121 we can devide the category into 5 , each containing 5
items. Also try to arrange it alphabetically.

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/indian-spices.json]: @shahbaz17 Alphabetically is good enough :+1: - just a way for people to easily find what they are looking for.

Maybe the categories could be the letter ranges? E.g., `A-C`, `D-F`, `G-P` etc.

**Comment by shahbaz17** [comment]
@GuiltyDolphin Alphabetical category will do.
Hello @hiteshaswani121 , just rename the category and each category should contain atleast 5 item.

**Comment by shahbaz17** [comment]
Hello @hiteshaswani121 
Your input required. R u working on it ?
Or else should i do the suggested change ?

**Comment by moollaza** [comment]
@hiteshaswani121 @shahbaz17 this looks good for now.

We should definitely organize them alphabetically next though. Would either of you like to do that? If not please make an issue for the community to do it.

**Comment by shahbaz17** [comment]
@moollaza I will do the suggested change.

**Comment by shahbaz17** [comment]
https://github.com/duckduckgo/zeroclickinfo-goodies/pull/3174

**Pull request merged**